styles/npm/catppuccin.user.css

```diff
@@ -1098,19 +1098,28 @@
       }
     }
 
+    /* pricing page */
     .dark-gray {
         color: @text !important;
     }
 
     .bg-dark-gray, .a11y-light-gray, .bg-gray {
-        background-color: @accent-color !important;
+        background-color: @mauve !important;
```

Comment:
alright

Reply:
As in the Discord, please just leave a thumbs up instead of a full comment. Click the re-request review button in the sidebar once the changes have been implemented, or add a comment to clarify if need be.

Reply:
> As in the Discord, please just leave a thumbs up instead of a full comment. Click the re-request review button in the sidebar once the changes have been implemented, or add a comment to clarify if need be.

also @uncenter what if i can dm you on discord so you and me dont have to leave a bunch of github comments on a converstation?

Reply:
I'd prefer not. GitHub comments are useful for what they are intended for, which is exactly this.